sdk/python/feast/templates/aws/feature_repo/feature_store.yaml

```diff
@@ -0,0 +1,27 @@
+project: my_project
+# By default, the registry is a file (but can be turned into a more scalable SQL-backed registry)
+# On AWS, minimally you should create a S3 bucket for a remote file registry
+registry: data/registry.db
+# The provider primarily specifies default offline / online stores & storing the registry in a given cloud
+provider: aws
+# Note: if you comment the online store paths out, then this will materialize to Datastore
+online_store:
+    type: sqlite
+    path: data/online_store.db
+# See https://docs.feast.dev/reference/online-stores/dynamodb
+#online_store:
```

Comment:
did you mean to include datastore as well?

Reply:
no? we don't support datastore on aws?